auraed/src/runtime/free_cell.rs

```diff
@@ -0,0 +1,27 @@
+use crate::runtime::cell_name::CellName;
+use crate::runtime::CellService;
+use aurae_proto::runtime::{FreeCellRequest, FreeCellResponse};
+use tonic::Status;
+use validation_macros::ValidatedType;
+
+#[derive(ValidatedType)]
+pub(crate) struct ValidatedFreeCellRequest {
+    #[field_type(String)]
+    #[validate]
+    pub cell_name: CellName,
+}
+
+impl FreeCellRequestTypeValidator for FreeCellRequestValidator {}
```

Comment:
this line makes my head hurt.  i don't know what either of these types are or why we'd need an empty impl of one for the other..

Reply:
This won't always be empty. If the trait `[WHATEVER]TypeValidator` has any required functions, the compiler will complain. In this case, marking the field `#[validate]` tells the macro to generate the default function for validation.